tests/bam_io/test_bam_file.h

```diff
@@ -480,14 +480,14 @@ SEQAN_DEFINE_TEST(test_bam_io_bam_file_bam_file_seek)
     seqan::BamHeader header;
     readHeader(header, bamFile);
 
-    seqan::String<seqan::Pair<off_t, int> > recs;
+    seqan::String<seqan::Pair<uint64_t, int> > recs;
```

Comment:
Ok, I changed it to the Position<BamFile>::Type type, which is the same type returned by position().